twill-api/src/main/java/org/apache/twill/api/TwillPreparer.java

```diff
@@ -257,8 +258,19 @@
   TwillPreparer setLogLevels(String runnableName, Map<String, LogEntry.Level> logLevelsForRunnable);
 
   /**
-   * Starts the application.
+   * Starts the application. It's the same as calling {@link #start(long, TimeUnit)} with timeout of 60 seconds.
```

Comment:
There is, however it's defined in the core module, hence can't refer to it in the API module